src/main/scala/cromwell/engine/db/slick/SymbolComponent.scala

```diff
@@ -93,4 +96,12 @@ trait SymbolComponent {
       workflowExecution <- symbol.workflowExecution
       if workflowExecution.workflowExecutionUuid === workflowExecutionUuid
     } yield symbol)
+
+  val symbolsForWorkflowOutput = Compiled(
+    (workflowExecutionUuid: Rep[String]) => for {
+      symbol <- symbols
+      if symbol.index === -1 && symbol.reportableResult === true
```

Comment:
It seems strange to dereference a dynamic scatter list from a static WDL file.

I think I'd want to see a solid use case before enabling that.